dsgrid/config/dimensions.py

```diff
@@ -49,6 +49,21 @@ class DimensionBaseModel(DSGBaseModel):
         ),
         updateable=False,
     )
+    display_name: str = Field(
+        title="display_name",
+        description="Display name. Source for auto-generated query_name.",
+        note="Dimension display names should be singular, concise names that are common across "
+        "projects and datasets. Must be unique within a project or dataset.",
+        notes=(
+            "Only alphanumeric characters and spaces are supported (no dashes or special characters).",
```

Comment:
Why spaces here? Aren't underscores better? If used in query, we'll always have to wrap names in "" in sql if with spaces.

Reply:
" Symbols like *, & and @ are also considered alphanumeric characters." Do we want to support common symbols too?

Reply:
I'm not sure those characters are generally considered alphanumeric. I can just add "and underscores" to remove ambiguity.